Tomlet.Tests/TableTests.cs

```diff
@@ -46,6 +46,26 @@ public void TablesCanHaveQuotedKeyNames()
             var tomlString = document.SerializedValue.Trim();
             Assert.Equal(inputString, tomlString);
         }
+        
+         [Theory]
+         [InlineData("normal-key", "normal-key")]
+         [InlineData("normal_key", "normal_key")]
+         [InlineData("normalkey", "normalkey")]
+         [InlineData("\"key with spaces\"", "\"key with spaces\"")]
+         [InlineData("key!with{}(*%&)random[other+symbols", "\"key!with{}(*%&)random[other+symbols\"")]
+         [InlineData("key/with/slashes", "\"key/with/slashes\"")]
+         [InlineData("Nam\\e", "\"Nam\\\\e\"")]
+         public void KeysShouldBeSerializedCorrectly(string inputKey, string expectedKey)
+         {
+             var inputString = $"""
```

Comment:
Raw string literals aren't supported on the current language version/net6

Reply:
Can we move the tests project to .NET 7?

Reply:
I suppose that's an option yeah. Let me get on that

Reply:
Alright, if you rebase onto master now it should work